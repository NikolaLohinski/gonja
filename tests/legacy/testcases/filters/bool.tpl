"true": {{ "true" | bool }}
"True": {{ "True" | bool }}
"yes": {{ "yes" | bool }}
"Yes": {{ "Yes" | bool }}
"on": {{ "on" | bool }}
"On": {{ "On" | bool }}
"TrUe": {{ "TrUe" | bool }}
"1": {{ "1" | bool }}
"false": {{ "false" | bool }}
"False": {{ "False" | bool }}
"no": {{ "no" | bool }}
"No": {{ "No" | bool }}
"off": {{ "off" | bool }}
"Off": {{ "Off" | bool }}
"FaLSe": {{ "FaLSe" | bool }}
"0": {{ "0" | bool }}
"": {{ "" | bool }}
1: {{ 1 | bool }}
1.0: {{ 1.0 | bool }}
0: {{ 0 | bool }}
0.0: {{ 0 | bool }}
False: {{ False | bool }}
True: {{ True | bool }}
false: {{ false | bool }}
true: {{ true | bool }}
nil: {{ nil | bool }}
None: {{ None | bool }}