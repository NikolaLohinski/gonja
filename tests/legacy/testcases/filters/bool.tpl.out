"true": True
"True": True
"yes": True
"Yes": True
"on": True
"On": True
"TrUe": True
"1": True
"false": False
"False": False
"no": False
"No": False
"off": False
"Off": False
"FaLSe": False
"0": False
"": False
1: True
1.0: True
0: False
0.0: False
False: False
True: True
false: False
true: True
nil: False
None: False